knowledge-base/set-culture-of-date-picker-in-angular-viewer.md

```diff
@@ -17,66 +17,68 @@ res_type: kb
+		</tr>
 	</tbody>
 </table>
 
 
 ## Description
 The article describes how to change the culture of the Kendo DatePicker widget used in the Angular Report Viewer.
 
+The general solution of the problem for localizing the DatePicker of the [pure Html5 Report Viewer](https://docs.telerik.com/reporting/html5-report-viewer) is described in the [Change the culture of Kendo DatePicker widget in the parameters area of the HTML5 Report Viewer]({% slug how-to-change-the-culture-of-the-datepicker-in-html5-report-viewer %}) article. 
+
+To also change the culture of the report viewer, it is required to import the necessary Kendo culture on the page with the report viewer.
+
 ## Solution
-The general solution of the problem for localizing the DatePicker of the 
-[pure Html5 Report Viewer](https://docs.telerik.com/reporting/html5-report-viewer) is described in the 
-[Change the culture of Kendo DatePicker widget in the parameters area of the HTML5 Report Viewer]({% slug how-to-change-the-culture-of-the-datepicker-in-html5-report-viewer %}) 
-article. The Angular viewer is a wrapper of the pure Html5 viewer that utilizes Kendo UI for jQuery widgets. For that reason, 
-in order to update the DatePicker in the Angular Viewer according to the above article, you need first to integrate the 
-Kendo UI for jQuery in the Angular application following the article 
-[Kendo UI for jQuery Integration](https://www.telerik.com/kendo-angular-ui/components/framework/kendo-jquery/). Here are the steps: 
 
-- Install Kendo UI for jQuery:
+For this example, we will load the additional `kendo` cultures via the kendo CDN. To accomplish this in an Angular component, it is necessary to create a helper function that loads scripts from an URL, for example:
+
+```TypeScript
+    loadScript(url: string) {
+        let body = <HTMLDivElement>document.body;
+        let script = document.createElement('script');
+        script.innerHTML = '';
+        script.src = url;
+        script.async = true;
+        script.defer = true;
+        body.appendChild(script);
+    }
 ```
-npm install --save @progress/kendo-ui
-``` 
 
- - In the 'app.component.ts' import the components from Kendo and declare the 'kendo' object. Then in the 'AfterViewInit' add 
- the code for utilizing the corresponding culture:
+The script can be loaded in the `constructor` of the component loading the Angular Report Viewer, for example:
+
 ```TypeScript
-...
-import '@progress/kendo-ui';
-declare var kendo: any;
-...
-export class AppComponent implements AfterViewInit {
-    @ViewChild('viewer1', { static: false }) viewer: TelerikReportViewerComponent;
+export class ReportViewerComponent implements AfterViewInit {
+    @ViewChild('viewer1') viewer: TelerikReportViewerComponent;
 
-    ngAfterViewInit(): void {
-        ...
-        kendo.culture("fr-FR");
-		...
+    constructor() {
+        this.loadScript(`http://kendo.cdn.telerik.com/2022.3.913/js/cultures/kendo.culture.bg-BG.min.js`);
```

Comment:
Consider using the variable form _config.yaml {{site.kendosubsetversion}} rather than hardcoded version